src/main/java/com/beerjournal/breweriana/user/persistence/UserRepository.java

```diff
@@ -31,6 +35,21 @@
         return crudRepository.findOneByEmail(email);
     }
 
+    public User deleteOneById(ObjectId objectId) {
+        //// TODO: 2017-04-28 should we also delete usercollection/items?
```

Comment:
I decided to delete user collection (as it's not much work), and leave items that were created for now